deploy-manage/deploy/cloud-on-k8s/configuration-standalone.md

```diff
@@ -269,6 +269,62 @@ To deploy Elastic Agent in clusters with the Pod Security Policy admission contr
 
 In order to run {{agent}} as a non-root user you must choose how you want to persist data to the Agent’s volume.
```

Comment:
are these procedures the exact same across this page and the `configuration-fleet` page? if so, we should make this a snippet that can be reused so when the procedure is updated in one place, it can be reflected in the other place.